src/ixdat/techniques/reactor.py

```diff
@@ -0,0 +1,289 @@
+                "measurement.clear_cache(). Using original unit {original_unit} !",
+                stacklevel=2,
+            )
+            unit_factor = 1
+            new_unit = original_unit
+        return unit_factor, new_unit
+
+
+class SpectroReactorMeasurement(ReactorMeasurement, SpectroMSMeasurement):
+    default_plotter = SpectroTPMSPlotter
+
+
+class ReactorUnitConverterResult(Saveable):
+    """A class for a reactor reactor_calibration result.
+
+    FIXME: I think that something inheriting directly from Saveable does not belong in
+        a technique module.
+    """
+
+    table_name = "reactor_cal_results"
+    column_attrs = {"name", "mass", "new_unit"}
+
+    def __init__(
+        self,
+        name=None,
+        mass=None,
+        new_unit=None,
+    ):
+        super().__init__()
+        self.name = name or f"{mass}@{new_unit}"
+        self.mass = mass
+        self.new_unit = new_unit
+
+    def __repr__(self):
+        return (
+            f"{self.__class__.__name__}(name={self.name}, mol={self.mass}, "
+            f"calibration type={self.cal_type})"
+        )
+
+    # @property
+    # def color(self):
+    #    return STANDARD_COLORS[self.mass]
+
+
+class ReactorCalibration(Calibration):
+    """A reactor calibration to calibrate inverse of meta_series"""
+
+    extra_column_attrs = {
+        "reactor_calibration_results": ("reactor_cal_results", "reactor_cal_result_id")
+    }
+    # TODO: https://github.com/ixdat/ixdat/pull/11#discussion_r677552828
+
+    def __init__(
+        self,
+        name=None,
+        date=None,
+        tstamp=None,
+        setup=None,
+        reactor_cal_results=None,
+        technique="reactor",
+        new_unit=None,
+        measurement=None,
+    ):
+        """Initiate a Calibration
+
+        Args:
+            name (str): The name of the reactor_calibration
+            date (str): Date of the reactor_calibration
+            setup (str): Setup where the measurmenet was done
+            technique (str): The technique of the calibration
+            tstamp (float): The time at which the calibration took place or is valid
+            measurement (TPMSMeasurement): Optional.A measurement to calibrate by default
+        """
+        super().__init__(
+            name=name or f"TPMS reactor_calibration for {setup} on {date}",
+            technique=technique,
+            tstamp=tstamp,
+            measurement=measurement,
+        )
+
+        self.name = name
+        self.date = date
+        self.setup = setup
+        self.new_unit = new_unit
+
+        # self.reactor_unit_convert = reactor_unit_results or []
+
+    def __repr__(self):
+        # TODO: make __repr__'s consistent throught ixdat
+        return (
+            f"{self.__class__.__name__}"
+            f"(Calibration={self.name} for setup={self.setup} on date={self.date})"
+        )
+
+    def calibrate_series(self, key, measurement=None):
```

Comment:
I think this is a fine use of a Calibration, to make sure "inverse_temperature" is available for arrhenius plot. There are other more obvious ways to do it, but I personally think it's elegant :) 

But **where does the calibration get added to a `ReactorMeasurement`**?

Reply:
At the moment I add the ms_calibration and reactor_calibration after meas.read(cinf_db_reader)

''' Add calibrations '''

#Add MS calibration for QMS
meas.add_calibration(ms_calibration)
# reactor calibration to get inverse of dataseries
meas.add_calibration(ReactorCalibration())

Reply:
The calibration has been added in the init of the class

Reply:
I think this is better as a ReactorMeasurement will need this calibration wherever it is read from.